code/modules/mob/living/basic/snails/snail_ai.dm

```diff
@@ -0,0 +1,43 @@
+/datum/ai_controller/basic_controller/snail
+	blackboard = list(
+		BB_TARGETING_STRATEGY = /datum/targeting_strategy/basic,
+	)
+
+	ai_movement = /datum/ai_movement/basic_avoidance
+	idle_behavior = /datum/idle_behavior/idle_random_walk
+	planning_subtrees = list(
+		/datum/ai_planning_subtree/find_nearest_thing_which_attacked_me_to_flee,
+		/datum/ai_planning_subtree/flee_target,
+		/datum/ai_planning_subtree/find_food,
+		/datum/ai_planning_subtree/find_and_hunt_target/snail_people,
+	)
+
+/datum/ai_planning_subtree/find_and_hunt_target/snail_people
+	target_key = BB_SNAIL_PERSON
```

Comment:
Do we actually need a unique key for this when the snails don't have any other hunt behaviours?